scss_test.scss

```diff
@@ -0,0 +1,3 @@
+.parent {
+  background: #ffffff;
```

Comment:
Color literals like `#ffffff` should only be used in variable declarations; they should be referred to via variable everywhere else.